### DISCUSSION TITLE
`main` or `master` as head branch name?

### DISCUSSION THREAD

**Discussion by andrey-helldar** [Laravel-Lang/common#24] [Q&A] [answered]
@Laravel-Lang/laravel-lang, what do you think is the best version of the head branch name?

In my opinion, the use of the "master/slave" combination is as old as the IDE HDD. This is a relic of the past.

Personally, I like to use the word "main" more and I started doing this long before it became mainstream.

What do you think about this?

**Comment by mohamedsabil83** [comment, marked as answer]
I agreed 👍🏽 Like you, It's been my approach for a while now and, as you said, it became the mainstream

**Comment by caouecs** [comment]
ok for main

**Comment by andrey-helldar** [comment]
Ready 😎

**Comment by luisprmat** [comment]
It should be *main* branch
